src/main.rs

```diff
@@ -381,26 +376,14 @@ impl Fixer {
         db_dir.push(&syms_dir);
         db_dir.push(&db_seg);
 
-        // - Unix: `db_entries` iterates over `syms/libxul.so/`
-        // - Windows: `db_entries` iterates over `syms/xul.pdb/`
-        let mut db_entries = fs::read_dir(&db_dir)
-            .context(
-                "note: this is expected and harmless for system libraries on debug automation runs",
```

Comment:
This is a useful error message on automation and I would like to preserve it. I think it needs to be moved to the `fs::read(&sym_file)` call below.

(While you're there, can you change the "read symbols file" on that error message to "read breakpad symbols file", for consistency with the other error messages in this function. Thanks.)